callback.php

```diff
@@ -92,6 +92,12 @@
                     $o->setCurrentState(Configuration::get('PS_OS_PAYMENT'));
                 }
             }
+        } else {
+            echo 'Order not found';
+            return;
```

Comment:
An empty return is not required as there is no calling function

Reply:
True, removed the empty return statements.